src/helm/proxy/clients/anthropic_client.py

```diff
@@ -18,13 +19,145 @@
 from .client import Client, wrap_request_time, truncate_sequence
 
 
+class AnthropicClient(Client):
+    """
+    Client for the Anthropic models (https://arxiv.org/abs/2204.05862).
+    They use their own version of the GPT-2 tokenizer.
+    """
+
+    MAX_COMPLETION_LENGTH: int = 8192
+    ADDITIONAL_TOKENS: int = 5
+    PROMPT_ANSWER_START: str = "The answer is"
+
+    def __init__(self, api_key: str, cache_config: CacheConfig, tokenizer_client: Client):
+        self.api_key: str = api_key
+        self.cache = Cache(cache_config)
+        self.tokenizer_client: Client = tokenizer_client
+        self._client = anthropic.Client(api_key)
+
+    def _send_request(self, raw_request: Dict[str, Any]) -> Dict[str, Any]:
+        result = self._client.completion(**raw_request)
+        assert "error" not in result, f"Request failed with error: {result['error']}"
+        return result
+
+    def _filter_completion(self, completion: str, max_tokens: int) -> str:
+        # If the completion starts with a colon, space, or newline, remove it.
+        for _ in range(AnthropicClient.ADDITIONAL_TOKENS):
```

Comment:
So I started to do this in `truncate_sequence` but I am not sure it is the right place to do this. In fact, this is how a call looks like:
- We get the completion from the API
- We filter the completion
- We tokenize it
- We truncate the sequence.
If we wanted to add this filtering in `truncate_sequence` then it means that the completion would have to be re-tokenized.
Also since this is Anthropic specific should this really be in `client.py`?

Reply:
`truncate_sequence` also [truncates the tokens](https://github.com/stanford-crfm/helm/blob/c36c5f900de99e2e26418af3923ce7dffecdd74f/src/helm/proxy/clients/client.py#L80-L93), so you shouldn't need to retokenize after truncating.

What is the conclusion here? I think the code and comments are fine as is, but I would prefer to _enable_ the warnings, because clearly the warnings are warning us of _actual_ bugs in Anthropic.

Reply:
Ok you're right. I have reenabled warnings